schema.go

```diff
@@ -1045,18 +1074,20 @@ func (s *Schema) genSingleIndexQuery(t *Table, g *Generator, r *rand.Rand, p Par
 		return nil
 	}
 
-	/* Once we have ALLOW FILTERING SUPPORT this can be applied
-	pkNum := p.Rand.Intn(len(t.PartitionKeys))
+	 // Once we have ALLOW FILTERING SUPPORT this can be applied
+	pkNum := r.Intn(len(t.Indexes))
```

Comment:
how is this `pkNum` used?

Reply:
Fixed, i think it is used to randomly get number of indexes  for query from 1 to len(Indexes)

Reply:
I mean, I don't see where this variable is used?

Reply:
yep. Now it is used )